src/main/java/com/evilcorp/bloom/repo/CategoryRepo.java

```diff
@@ -0,0 +1,23 @@
+package com.evilcorp.bloom.repo;
+
+import java.util.Optional;
+import java.util.UUID;
+
+import org.springframework.data.jdbc.repository.query.Modifying;
+import org.springframework.data.jdbc.repository.query.Query;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.data.repository.query.Param;
+import org.springframework.stereotype.Repository;
+
+import com.evilcorp.bloom.model.Category;
+
+@Repository
+public interface CategoryRepo extends CrudRepository<Category, UUID> {
+  Optional<Category> findByName(String name);
+
+  boolean existsByName(String name);
```

Comment:
It is good to create index for this frequent access patterns that you have

Will you have scenarios where you will call exist and follow up  with find or delete? If yes, you should consider to call just find or delete and handle the scenario where these commands doesn't return anything

The benefit is that it will reduce a network call, less call hitting your database and it will simplify the code